default/python/universe_generation/universe_tables.py

```diff
@@ -141,6 +141,7 @@
 }
 
 # The maximum length of any starlane the galaxy generator can create (in uu).
+# MAX_STARLANE_LENGTH >= 2 * MIN_SYSTEM_SEPARATION
```

Comment:
`stitching_positions()` only works if there is at least 2*MIN_SYSTEM_SEPARATION between the nearest stars in two groups, in order to generate a position for the extra star(s).  I put the comment here so that it is right at the location where someone will change the MAX_STARLANE_LENGTH.